analytics/utils.py

```diff
@@ -0,0 +1,53 @@
+from bs4 import BeautifulSoup
+
+from django.conf import settings
+from django.utils import timezone
+
+from google_analytics.utils import get_visitor_id
+
+
+def should_ignore(request):
+    if hasattr(settings, 'ANALYTICS_IGNORE_PATH'):
+        exclude = [p for p in settings.ANALYTICS_IGNORE_PATH
+                   if request.path.startswith(p)]
+        return any(exclude)
+    return False
```

Comment:
This will also include page views for people who didn't go through the analytics endpoint. Just want to make sure that that is the desired functionality.

Reply:
Yep, this was intended